### ISSUE TITLE
duplicated help_block on select/choice

### ISSUE THREAD

**Issue by mix359** [kristijanhusak/laravel-form-builder#153]
The help_block on a select or choice, is inserted two times under the field.

Here's and example of code that create the problem:
```php
$form->add("field_id", "choice", [
			'label' => "Title",
			'default_value' => "",
			'choices' => ['aaa','bbb'],
			'multiple' => true,
			'help_block' => ['text' => "Sample text", 'tag' => 'p', 'attr' => ['class' => 'help-block']]
		]);
```
This produces two <p> elements under the field

I'm using the last version (1.6.20)

**Comment by kristijanhusak** [comment]
@mix359 can you try using `dev-master` and see if it happens?

**Comment by kristijanhusak** [comment]
I tested it, and the bug exists there also. I fixed it in the meantime, so just pull latest `dev-master` and it will work out. I will publish stable version soon.

**Note**: If you published views make sure you update them accordingly to last commit.

**Comment by kristijanhusak** [comment]
@mix359 Fixed in latest release (1.6.30). Thanks for reporting.
